src/banner/banner.js

```diff
@@ -34,18 +37,18 @@ export default class Banner {
                 let secondaryLinkList = this.createSecondaryLinks(secondaryLinks)
                 secondaryElement = newElement('div', {
                     children: secondaryLinkList,
-                    classes: `${styling.linkWrapper} ${this.theme.linkWrapper}`
+                    classes: `${this.styling.linkWrapper} ${this.theme.linkWrapper}`
```

Comment:
I believe this should be 
```
classes: `${this.theme.linkWrapper} ${this.styling.linkWrapper}`
```
so that custom styles are last and overwrite the theme.

Reply:
yes that should be changed, thank you.

Reply:
Given your use case and future changes that people may use this functionality for